include/tachimawari/control/node/control_node.hpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2021 Ichiro ITS
```

Comment:
Please update the copyright year in all relevant files, including the `LICENSE` file.
```suggestion
// Copyright (c) [start-year]-2023 Ichiro ITS
```
Replace `[start-year]` with the year when this project started.